src/oogeso/core/networks/edge.py

```diff
@@ -99,9 +101,12 @@ def _loss_function_constraint(self, i: int, pyomo_model: pyo.Model, piecewise_re
         pyomo_model.varEdgeFlow21[self.id, :].setub(self.edge_data.flow_max)
         # Losses on cables are: P_loss = R/V^2 * P^2, i.e. quadratic function of power flow
         # Losses in transformers are: P_loss = ...
-        lookup_table = self.edge_data.power_loss_function
-        pw_x = lookup_table[0]
-        pw_y = lookup_table[1]
+        if hasattr(self.edge_data, "power_loss_function"):
+            lookup_table = self.edge_data.power_loss_function  # Fixme: What do we do if there are no loss functions?
+            pw_x = lookup_table[0]
+            pw_y = lookup_table[1]
+        else:  # Fixme: Only certain Edges expects loss function... Should we default power loss function to 0, 0?
```

Comment:
Could this happen? How do we want to handle this? It seems like the code expect a certain type of edge that has a loss function. But not all of them does? Or should we implement this is in the base class?